src/common/address.js

```diff
@@ -384,6 +386,19 @@ const states = {
     .sort((stateA, stateB) => stateA.label.localeCompare(stateB.label)),
 };
 
+const buildStatesWithFullCountryNames = () => {
+  const statesCopy = _.cloneDeep(states);
+  statesCopy.Canada = statesCopy.CAN;
+  statesCopy['United States'] = statesCopy.USA;
+  statesCopy.Mexico = statesCopy.MEX;
+  delete statesCopy.CAN;
+  delete statesCopy.USA;
+  delete statesCopy.MEX;
+  return statesCopy;
+};
+
+const statesWithFullCountryNames = buildStatesWithFullCountryNames();
```

Comment:
It seems to me that this could be solved on the front-end with some creative labeling... adding a full list of `statesWithFullCountryNames` duplicates a lot of content in the `constants.json` making it bigger than it needs to be.

Reply:
The reason for this function and `addressBuilder` is because our backend is providing us full country names and the current `address` common definition is looking for the abbreviations of the countries:

```
const address = (() => {
  // eslint-disable-next-line import/no-named-as-default-member
  const countries = constants.countries.map(object => object.value);
...
})();```

Reply:
That's fine, but it shouldn't need to be added to the `constants.json`

Reply:
I was following suit with how `states` was being imported and exported but I can provide a direct reference from `address.js`.

Reply:
Just pushed up a refactor.